inventory-template.ini

```diff
@@ -15,7 +15,7 @@ deployment_type=openshift-enterprise
 
 openshift_docker_additional_registries={additional_registries}
 openshift_docker_insecure_registries={additional_registries}
-openshift_docker_blocked_registries=registry.hacker.com
+openshift_docker_blocked_registries=
```

Comment:
It'd probably be better to remove this line entirely.

Reply:
done.
I've confirmed that absense of this line is OK (well not surprising) when I installed a very trimmed down inventory.